dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task222208CreateVariantTableTest.java

```diff
@@ -0,0 +1,89 @@
+package com.dotmarketing.startup.runonce;
+
+import static com.dotcms.util.CollectionsUtils.map;
+import static graphql.Assert.assertTrue;
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertFalse;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.common.db.DotDatabaseMetaData;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.db.DbConnectionFactory.DataBaseType;
+import com.dotmarketing.exception.DotDataException;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class Task222208CreateVariantTableTest {
```

Comment:
Test needs to be added to MainSuite

Reply:
done https://github.com/dotCMS/core/pull/22827/commits/280f4a175649315b2ac562270104d588598764d7